src/engraving/rw/write/twrite.cpp

```diff
@@ -910,7 +910,7 @@ void TWrite::write(const Chord* item, XmlWriter& xml, WriteContext& ctx)
     if (item->hook() && item->hook()->isUserModified()) {
         write(item->hook(), xml, ctx);
     }
-    if (item->showStemSlash() && item->isUserModified()) {
+    if (item->showStemSlash() && item->showStemSlash() != item->propertyDefault(Pid::SHOW_STEM_SLASH).toBool()) {
```

Comment:
As part of #28118 I've found myself doing kind of the opposite of this, i.e. override/specialize isUserModified as necessary to include this kind of information. In principle, I'd find it preferable for the read/write modules to know as little as possible about the inside working of each EngravingItem, so I think this kind of information would be better hidden inside the isUserModified method. But it's just a thing of principle so feel free to also do this in a next PR